### ISSUE TITLE
Unable to bring up K8S single node setup

### ISSUE THREAD

**Issue by pavanbhat21** [projectatomic/adb-atomic-developer-bundle#429]
Unable to bring up the K8S single node VM, it errors out as below:

[pavan@sgi-xe240-01 rhel-k8s-singlenode-setup]$ vagrant up
Bringing machine 'default' up with 'libvirt' provider...
==> default: Creating image (snapshot of base box volume).
==> default: Creating domain with the following settings...
==> default:  -- Name:              rhel-k8s-singlenode-setup_default
==> default:  -- Domain type:       kvm
==> default:  -- Cpus:              2
==> default:  -- Memory:            1024M
==> default:  -- Base box:          cdkv2
==> default:  -- Storage pool:      default
==> default:  -- Image:             /var/lib/libvirt/images/rhel-k8s-singlenode-setup_default.img
==> default:  -- Volume Cache:      default
==> default:  -- Kernel:  
==> default:  -- Initrd:  
==> default:  -- Graphics Type:     vnc
==> default:  -- Graphics Port:     5900
==> default:  -- Graphics IP:       127.0.0.1
==> default:  -- Graphics Password: Not defined
==> default:  -- Video Type:        cirrus
==> default:  -- Video VRAM:        9216
==> default:  -- Keymap:            en-us
==> default:  -- Command line : 
==> default: Creating shared folders metadata...
Error while setting up autostart on network: undefined method `autostart=' for nil:NilClass.
[pavan@sgi-xe240-01 rhel-k8s-singlenode-setup]$ 

This probably is a blocker.

**Comment by pavanbhat21** [comment]
Note that this behaviour is observed on RHEL

**Comment by optak** [comment]
I tried to reconfigure Vagrantfile to use static IP and it works. Not sure about possible side effects.
If you choice this solution, make sure it's different IP than in rhel-ose Vagrantfile

**Comment by LalatenduMohanty** [comment]
This issue was reported in upstream vagrant-libvirt https://github.com/vagrant-libvirt/vagrant-libvirt/issues/427

**Comment by LalatenduMohanty** [comment]
@pavanbhat21 What is version of vagrant-libvirt plugin you are using? This was present before version 0.0.31 I think.

**Comment by optak** [comment]
It looks that it's already fixed in upstream. But can we have update of sclo-vagrant1 collection for CDK-2.1?
If not, I would use static IP, unless there is some problem with static IP for k8s.

**Comment by pavanbhat21** [comment]
Details below:

```
[root@sgi ~]# cat /etc/redhat-release 
Red Hat Enterprise Linux Workstation release 7.2 (Maipo)

[pavan@sgi ~]$ vagrant plugin list
vagrant-libvirt (0.0.30, system)
vagrant-registration (1.2.1)
  - Version Constraint: 1.2.1
vagrant-service-manager (1.1.0.beta.1)
  - Version Constraint: 1.1.0.beta.1
vagrant-sshfs (1.1.0)
  - Version Constraint: 1.1.0


 Vagrant version used is 1.7.4
```

**Comment by LalatenduMohanty** [comment]
@pavanbhat21 @optak can we close this now? As the issue is resolved in 0.0.32

**Comment by optak** [comment]
Yes, 0.0.32 is in sclo-vagrant1 collection, so we can close this issue.
